fmt/format.cc

```diff
@@ -486,34 +483,6 @@ FMT_FUNC void report_windows_error(
 }
 #endif
 
-FMT_FUNC void print(std::FILE *f, CStringRef format_str, ArgList args) {
```

Comment:
This and the following two methods are actually part of the core library and not `printf` extension